java/com/fortech/internship/controller/CityController.java

```diff
@@ -0,0 +1,61 @@
+package com.fortech.internship.controller;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fortech.internship.dao.CityDAO;
+import com.fortech.internship.dao.CityDAOImpl;
+import com.fortech.internship.dto.CityDTO;
+import com.fortech.internship.model.City;
+import com.fortech.internship.service.impl.AddressService;
+import com.fortech.internship.service.impl.CityService;
+
+@RestController
+public class CityController {
+
+	//@Autowired
+	//private CityDAO cityDAOImpl;
+	
```

Comment:
the same comments as for AddressController can be applied here

Reply:
solved, please check